mautic/mautic
# Fingerprint2 library added to mtc.js and the pixel query contains some info from it

Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      | N
| New feature?  | Y
| BC breaks?    | N
| Deprecations? | N
| Fixed issues  |  /

## Description
This PR adds the https://github.com/Valve/fingerprintjs2 library to mtc.js. The fingerprint and some other useful information are sent back to Mautic in the tracking pixel query. The query contains this in my case:

```
page_title:Test of the JS
page_language:en-US
page_referrer:
page_url:http://localhost/mauticjstest.html
email:my@gmail.com
fingerprint:9309a31e25389d242376f438abb310a0
resolution:1920x1080
timezone_offset:-120
platform:Linux x86_64
do_not_track:unknown
adblock:false
```

## Steps to test this PR
Testing is the same as at https://github.com/mautic/mautic/pull/1689. You can create the contact fields to store the data. Or just check the contact timeline that the data are visible in the page hit action details.
